crates/subspace-service/src/lib.rs

```diff
@@ -481,6 +482,9 @@ where
     } = partial_components;
 
     let root_block_cache = RootBlockCache::new(client.clone());
+    // Enable the alternate block relay implementation if the CLI params request
+    // disabling the default path.
+    let enable_block_relay = !config.base.announce_block;
```

Comment:
This is indirect to check whether the subspace block relay is enabled, it's more straightforward to add `enable_subspace_block_relay` in `SubspaceConfiguration` and initialize it from `cli.enable_subspace_block_relay`, and then you can use `config.enable_subspace_block_relay` in the service.

Reply:
Done